doc/common/references.md

```diff
@@ -37,13 +38,69 @@
 
 
 <a name="1.2"></a>
-## RM References
+## Cloud Native and Kubernetes References
+
+|Ref	     | Doc Number	            |Title          |
+|--------  |------------------------|---------------|
+| | | "Extended Cloud Native Principles". Available at https://networking.cloud-native-principles.org/cloud-native-principles. |
+| | | "DANM". Available at https://github.com/nokia/danm. |
+| | | "Kubernetes Container Runtime Interface (CRI)". Available at https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/. |
+| | | "Multus". Available at https://github.com/intel/multus-cni. |
```

Comment:
```suggestion
| | | "Multus". Available at https://github.com/k8snetworkplumbingwg/multus-cni. |
```

Reply:
@petorre Thanks - Accepted. FYI, in RA2 [3.2.2 Container Networking Services](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter03.md#322-container-networking-services), in the bullets before Table 3-1, the link for Multus is to the intel link.